.github/workflows/call-report.yml

```diff
@@ -0,0 +1,72 @@
+name: "Test calling report.yml"
+
+on:
+  workflow_dispatch:
+    inputs:
+      which-gap:
+        description: 'Which Gap version'
+        default: 'master'
+        required: false
+        type: string
+
+jobs:
+  call-report:
+    name: "Call report.yml"
+    uses: ./.github/workflows/report.yml
+    with:
+      which-gap: ${{ github.event.inputs.which-gap }}
+
+  update-latest:
+    name: "Upload report"
+    runs-on: ubuntu-latest
+    needs: call-report
+    steps:
+      - uses: actions/checkout@v2
+
+      # warning/error in utils.py throw a syntax error with default version of python
+      - name: "Set up Python 3.7"
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
```

Comment:
Can you tell me what error you get exactly, and where, when you remove this? Perhaps point me at a CI log showing the error?